packages/headless/doc-parser/src/actions-resolver.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  ApiEntryPoint,
+  ApiItemKind,
+  ApiVariable,
+} from '@microsoft/api-extractor-model';
+import {DocNode, DocParamBlock} from '@microsoft/tsdoc';
+import {AnyEntity, Entity} from './entity';
+import {emitAsTsDoc} from './tsdoc-emitter';
+
+interface ActionsNamespace {
+  name: string;
+  members: Action[];
+}
+
+enum ActionKind {
+  Action = 'action',
+  AsyncAction = 'asyncThunk',
+  AnalyticsAction = 'analyticsAction',
```

Comment:
Although realistically there's only action creators and async thunks, I was thinking we could describe them in a more developer-friendly manner (less redux-like) and perhaps we could even describe analytics actions as being their own "kind" of action. Opinions?